hooks/post_gen_project.py

```diff
@@ -2,7 +2,7 @@
 import os
 import shutil
 
-from path import Path
+from phathlib import Path
```

Comment:
I guess it shall be `pathlib`. 

Second, I'm not sure that all APIs coming from `path` are actually provided in `pathlib`. 
I remember I've met issues with pathlib, but don't remember with of them...

Reply:
Maybe you can write a test to see if anything pops up?

Reply:
@bneijt could you also try adding a unit test for this file? would increase chances of it getting merged.